rowan/components/scripts/deploymentPostload.st

```diff
@@ -0,0 +1,38 @@
+"Recompile LaunchpadApplication and CurrentlyRunningLaunchpadApplication with dbTransient option
+ so that instances are not persistent because they store session-dependent information only"
+
+| removeNonTransientVersionsOf |
+removeNonTransientVersionsOf := [:class |
+    | nonTransientVersions |
+    nonTransientVersions := class classHistory reject: [:classVersion | classVersion instancesDbTransient].
+    nonTransientVersions do: [:classVersion | class classHistory removeVersion: classVersion]].
+
+LaunchpadApplication instancesDbTransient
+    ifFalse: [
+        Object subclass: 'LaunchpadApplication'
+        	instVarNames: #( configurationCache configurationProvider mode commandServer )
+        	classVars: #()
+        	classInstVars: #()
+        	poolDictionaries: #()
+        	inDictionary: Launchpad
+        	options: #( dbTransient ).
+        (OrderedCollection new
+            add: LaunchpadApplication;
+            addAll: LaunchpadApplication allSubclasses;
+            yourself)
+                do: [:applicationClass | removeNonTransientVersionsOf value: applicationClass].
+    ].
```

Comment:
Couldn't we refer to the current definition of LaunchpadApplication and add the dbTransient option instead of hard coding the instance variable names, diccionary, superclass, etc?
Can we use something like withAllSubclasses to avoid building the collection manually?

Reply:
Addressed in 403e0be